docker/tensorflow-aarch64/Dockerfile

```diff
@@ -191,8 +191,8 @@ ENV PY_VERSION="${default_py_version}" \
     CPU="${cpu}" \
     ARCH="${arch}"
 # Key version numbers
-ENV NUMPY_VERSION=1.19.5 \
-    SCIPY_VERSION=1.5.2 \
+ENV NUMPY_VERSION=1.22.1 \
```

Comment:
I didn't encounter any problems using TensorFlow with these version numbers, but I think that's simply that the tests & examples don't have a SciPy dependency.
In any case, it is preferable to keep the version numbers across the TensorFlow and Torch builds consistent, so I would suggest opting for NumPy 1.21.5 here too,

Reply:
I agree with having the same version across pytorch and tensorflow.